web-apps/genesys-spark-examples/Jenkinsfile

```diff
@@ -13,7 +13,7 @@ webappPipeline {
     team = 'Core UI'
     mailer = 'matthew.cheely@genesys.com, daragh.king@genesys.com, jordan.stith@genesys.com, thomas.dillon@genesys.com, katie.bobbe@genesys.com, gavin.everett@genesys.com, jason.evans@genesys.com'
     chatGroupId = 'adhoc-30ab1aa8-d42e-4590-b2a4-c9f7cef6d51c'
-    nodeVersion = '18.16.1'
+    nodeVersion = '18.x multiarch'
```

Comment:
What's the multiarch bit for?

Reply:
More of a devops concern. It allows them to have one version of node that they can install on different machine types / architectures.